frontend/src/components/where-to-buy/where-to-buy.tsx

```diff
@@ -23,22 +23,33 @@ const WhereToBuy: React.FC<WhereToBuyProps> = ({
 }) => {
   const dispatch = useDispatch();
   const [isSorted, setIsSorted] = useState(false);
+  const [showSeeMore, setShowSeeMore] = useState(true);
   const { ads } = useAppSelector((state) => state.whereBuy);
-
-  const page = Number(
-    ads.find((ad) => ad.complectationId === complectationId)?.page,
-  );
-
+  const savedPage = ads.find(
+    (ad) => ad.complectationId === complectationId,
+  )?.page;
+  const page = savedPage ? savedPage : 0;
   const countpage = 20;
```

Comment:
Can you show only by 10 ads at a time? I think 20 ads is too much.

Reply:
done